upa/src/sdk/client.ts

```diff
@@ -139,3 +142,104 @@ export class UpaClient {
     return this.upaInstance.verifier.registerVK(vk.solidity());
   }
 }
+
+export type UnsignedOffChainSubmissionRequest = {
+  circuitIdProofAndInputs: application.CircuitIdProofAndInputs[];
+  submissionId: ethers.BytesLike;
+  expirationBlockNumber: bigint;
+  submitterNonce: bigint;
+  fee: bigint;
+  totalFee: bigint;
+};
+
+// Signed request to be sent out-of-band to an aggregator
+export type SignedOffChainSubmissionRequest = {
+  circuitIdProofAndInputs: application.CircuitIdProofAndInputs[];
+  submissionId: ethers.BytesLike;
+  expirationBlockNumber: bigint;
+  submitterNonce: bigint;
+  fee: bigint;
+  totalFee: bigint;
+  signature: string;
+};
```

Comment:
Is this just:
```suggestion
export type SignedOffChainSubmissionRequest = 
  UnsignedOffChainSubmissionRequest & 
  {  signature: string; };
```

Reply:
Yeah this is better. Though now that it's a class, I'd want `OffChainSubmissionRequest` to extend `UnsignedOffChainSubmissionRequest`, but it is already extending `OffChainSubmission`. Maybe we just store the whole `OffChainSubmissionRequest` in the DB to simplify the types by removing `OffChainSubmission`?